calico/openstack/mech_calico.py

```diff
@@ -344,7 +288,8 @@ def _ensure_socket_to_felix_peer(self, hostname, db_context):
                                                {'agent_type': AGENT_TYPE_FELIX,
                                                 'binary': '',
                                                 'host': hostname,
-                                                'topic': constants.L2_AGENT_TOPIC,
+                                                'topic':
+                                                constants.L2_AGENT_TOPIC,
```

Comment:
I would want to indent this one level to make it clearer that it's a value, not a key.